plugins/services/src/js/components/ServiceStatusWarningWithDebugInstruction.js

```diff
@@ -1,7 +1,7 @@
 import DateUtil from "#SRC/js/utils/DateUtil";
-import ServiceStatusWarning from "./ServiceStatusWarning";
+import ServiceStatusIcon from "./ServiceStatusIcon";
 
-class ServiceStatusWarningWithDebugInstruction extends ServiceStatusWarning {
+class ServiceStatusWarningWithDebugInstruction extends ServiceStatusIcon {
   getTooltipContent(timeWaiting) {
```

Comment:
I know this PR is WIP, but I don't understand how this works because this method is not in the superclass.

Reply:
Good question, but if you look at the code base today we have there.

I will take a look at it to make sure we are not using anywhere once I have everything else 100%